packages/ckeditor5-watchdog/src/editorwatchdog.ts

```diff
@@ -628,3 +673,22 @@ export type EditorWatchdogCreatorFunction<TEditor = Editor> = (
 	elementOrData: HTMLElement | string | Record<string, string> | Record<string, HTMLElement>,
 	config: EditorConfig
 ) => Promise<TEditor>;
+
+/**
+ * Returns editor-specific entries that can be used to identify whether an error context is connected to the editor.
+ */
+function getEditorSpecificEntries( editor: Editor ) {
+	return [
+		editor.data,
+		editor.plugins,
+		editor.config,
+		editor.conversion,
+		editor.commands,
+		editor.plugins,
+		editor.model.document,
+		editor.model,
+		editor.model.schema,
+		editor.ui?.view,
+		editor.editing?.view
+	].filter( Boolean );
```

Comment:
How did you select this list of objects? Maybe we should add some more like `editor.editing`, `editor.ui`, `editor.editing.mapper`?